script.js

```diff
@@ -26,6 +28,11 @@ function submitForm(event) {
   for (const answerInput of answerInputs) {
     answerInput.classList.remove('wrong-answer', 'correct-answer');
   }
+  //add message of submitting successfully
+  const messageEl = document.createElement('p');
+  messageEl.innerText = 'Question submitted successfully!';
+  messageEl.classList.add('message');
```

Comment:
Great to add this user feedback!

I would perhaps also "cleanup" afterwards. So once I have submitted a question and seen the Question submitted successfully! message then either a) remove message after X seconds or b) listen to on input enter on one of the fields to remove the message once the user starts to enter a new question they want to submit.

Reply:
Thank you for pointing that out! I added setTimeout to remove this element after 4 seconds.